src/components/r4-app.js

```diff
@@ -2,15 +2,17 @@ import {html, LitElement} from 'lit'
 import {ref, createRef} from 'lit/directives/ref.js'
 import {sdk} from '@radio4000/sdk'
 import page from 'page/page.mjs'
+import DatabaseListeners from '../libs/db-listeners'
 import '../pages/'
 
 export default class R4App extends LitElement {
 	playerRef = createRef()
 
 	static properties = {
-		/* public attributes, config props */
-		singleChannel: {type: Boolean, reflect: true, attribute: 'single-channel', state: true},
-		selectedSlug: {type: String, reflect: true, attribute: 'channel', state: true}, // channel slug
```

Comment:
@4www fyi these shouldn't have `state: true` as that will prevent Lit from converting the attributes into the properties we want.